src/app/components/DynamicContentStyles.tsx

```diff
@@ -9,8 +11,12 @@ import { AppServices } from '../types';
 import { assertDefined } from '../utils/assertions';
 
 // tslint:disable-next-line: variable-name
-export const WithStyles = styled.div`
-  ${(props: { styles: string }) => props.styles}
+export const ScopedGlobalStyle = createGlobalStyle`
```

Comment:
if you're using the styled global style, can you use the styled css processor here instead of postcss?

Reply:
good point